components/hab/src/command/pkg/channels.rs

```diff
@@ -41,11 +41,16 @@ use crate::{error::Result,
 /// # Failures
 ///
 /// * Fails if it cannot find the specified package in Builder.
-pub fn start(ui: &mut UI, bldr_url: &str, ident: &PackageIdent, token: Option<&str>) -> Result<()> {
+pub fn start(ui: &mut UI,
+             bldr_url: &str,
+             ident: &PackageIdent,
+             target: PackageTarget,
+             token: Option<&str>)
+             -> Result<()> {
     let api_client = Client::new(bldr_url, PRODUCT, VERSION, None)?;
 
     ui.begin(format!("Retrieving channels for {}", ident))?;
-    let channels = api_client.package_channels(ident, token)?;
+    let channels = api_client.package_channels(ident, target, token)?;
```

Comment:
Since "target" hasn't ever been a particularly obvious part of our API, we should add it to the `ui.begin(...)` line above.

(This applies to all modified commands... having clear feedback for _exactly_ which artifacts were operated on, particularly since we can have identical package identifiers apart from the target, is going to be very important.)

Reply:
Good point - will fix